woocommerce/woocommerce-android
# Added option to display existing images in Aztec

Fixes Step 2 of #1594. This PR adds the ability to display existing images Aztec editor.

### Testing
- Click on the product description in the product detail page for a product with an image in the description. Verify that the image is displayed.
- Click on the product description in the product detail page for a product without an image in the description. Verify that the existing functionality remains the same.

### Screenshots
<img width="300" src="https://user-images.githubusercontent.com/22608780/69410895-f26cda00-0d31-11ea-9e44-29b9b6d3ab06.png">

Update release notes:
